setup.cfg

```diff
@@ -1,3 +1,43 @@
+[metadata]
+name = fakeredis
+version = attr: fakeredis.__version__
```

Comment:
Interesting! I did not know about these directives, even though it is probably very old :-D